packages/ui/src/components/editor/properties/gltf/loader/index.tsx

```diff
@@ -147,15 +147,21 @@ const GLTFNodeEditor: EditorComponentType = (props) => {
       Icon={GLTFNodeEditor.iconComponent}
       {...props}
     >
-      <InputGroup name="Model Url" label={t('editor:properties.model.lbl-modelurl')}>
+      <InputGroup name="Model Url" className="flex flex-col gap-y-2" label={t('editor:properties.model.lbl-modelurl')}>
         <ModelInput
           value={gltfComponent.src.value}
           onRelease={(src) => {
+            if (src != gltfComponent.src.value) {
+              removeError(props.entity, GLTFComponent, 'LOADING_ERROR')
```

Comment:
removing errors like this should happen in the gltf component logic, not the UI

Reply:
Fixed!